src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -143,7 +143,7 @@ internal override async Task<IEnumerable<Diagnostic>> GetDiagnosticsAsync(
         AddAnalyzerToWorkspace(workspace, analyzer);
 
         var document = GetDocumentAndSelectSpan(workspace, out var span);
-        var allDiagnostics = await DiagnosticProviderTestUtilities.GetAllDiagnosticsAsync(workspace, document, span, includeNonLocalDocumentDiagnostics: parameters.includeNonLocalDocumentDiagnostics);
+        var allDiagnostics = await DiagnosticProviderTestUtilities.GetAllDiagnosticsAsync(workspace, document, span);
```

Comment:
tests were all over the map about what value they passed along.  Even though the value was often false, changing this to be true didn't break a single thing.  so tests were basically just passing along a value to satisfy the signature, with no one really knowing what value was appropriate.